### PR TITLE
Add sorting and filtering options, enhance accessibility 

### PR DESCRIPTION
**Pull request by brambrink**

### CODE REVIEW

File: src/components/logo/Logo.styled.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import styled from "styled-components";
 
-export const Logo = styled.a`
+export const Logo = styled.div`
   cursor: pointer;
```
Comment (RubenWerdmuller) [inline src/components/logo/Logo.styled.ts:4]: You changed this into a div, but the mouse still wants to point at it

File: src/components/menu-button/MenuButton.tsx [hunk 2]
```diff
@@ -15,7 +14,11 @@ export const MenuButton = ({ toggleMenu }: MenuButtonProps) => {
   const state = useContext(StoreContext);
   const [isMenuOpen] = state.isMenuOpen;
 
-  const icon = isMenuOpen ? <CrossIcon fill="#fff" width="14px" /> : <HamburgerIcon />;
+  const icon = isMenuOpen ? (
+    <Styled.CrossIcon arial-label="close" />
+  ) : (
+    <HamburgerIcon arial-label="menu" />
```
Comment (RubenWerdmuller) [inline src/components/menu-button/MenuButton.tsx:20]: `arial` isn't valid, is it?

File: src/components/time-entries/TimeEntries.tsx [hunk 3]
```diff
@@ -32,11 +32,13 @@ export const TimeEntries = ({ initialTimeEntries }: initialTimeEntriesProps) =>
 
   const state = useContext(StoreContext);
   const [timeEntries, setTimeEntries] = state.timeEntries;
-  const [isModalOpen, setIsModalOpen] = state.isModalOpen;
+  const [, setIsModalOpen] = state.isModalOpen;
 
   const [newTimeEntry, setNewTimeEntry] = useState({} as Partial<NewTimeEntryProps>);
   const [duration, setDuration] = useState("--:--");
 
+  const [clientFilter, setClientFilter] = useState("");
```
Comment (RubenWerdmuller) [inline src/components/time-entries/TimeEntries.tsx:40]: Remove empty line

**Review by RubenWerdmuller — CHANGES_REQUESTED** [review]

**Comment by RubenWerdmuller** [comment]
Nice busy! 👍

File: src/components/logo/Logo.styled.ts (hunk 1 above)
Comment (brambrink) [reply src/components/logo/Logo.styled.ts:4]: `  cursor: pointer;` makes sure the pointer is still pointing add it!

File: src/components/menu-button/MenuButton.tsx (hunk 2 above)
Comment (brambrink) [reply src/components/menu-button/MenuButton.tsx:20]: I get confused with the font, should be aria indeed

**Review by RubenWerdmuller — APPROVED** [review]

**Review by fabianovangool — APPROVED** [review]
Right on!

**Pull request merged**